### ISSUE TITLE
Track Follower for another rail numbers.

### ISSUE THREAD

**Issue by ginga81** [leezer3/OpenBVE#108]
Track Follower is incredibly surprised us.
But unfortunately, now version is only for Rail0.
I want to use the Track Follower for another rail.
Please consider about another rails.

**Comment by OnyxPhoenix** [comment]
There's still another way to keep "animated" trains following other tracks (even without track-follower) ¬¬

Still waiting for the multi/reverse-track capability as planned on Michelle's roadmap :(
That would be a nice improvement

**Comment by ginga81** [comment]
Track Follower system is an epochal system.
I thought that when I saw this system, how great system it is!
I cannot interpret code and build a program, sorry.

You said that it is already need to rewrite a lot of program.
I thought that when I saw this system, it is difficult to change to another track number.
So, I thought I have to say it early.

I thought that it should to change at the early time what is the Track Follower program using by original OpenBVE track base system(Rail X).
For example, if at the early time, it can put an another rail (for example only for Rail 15), at the future, if OpenBVE faces more complexity case, I thought that at the future, it is easy to rewrite program.

The current version of Track Follower, the animated object moves to parallel only.
The left-right axis and up-down axis moves difficult now.
Before hear OpenBVE 1.5, I thought that if the OpenBVE project is continuing, and if the case of required the Track Follower, I forecasted that OpenBVE uses something of another virtual route data file(for example XML).
Because, If a virtual moving object routine is dividing from main routine,  I thought it is perhaps easy to adding.
And I thought that if I can programming, I thought that the calculate and drawing program area will put at part of animated object routine.

At the current version of this system, if it is difficult to rewrite , I think that me too.

Track Follower function is already perfect.
So, I thought that if it can put to an another track number, it is more perfect.
